iphone/Classes/TiUIiOSStepper.m

```diff
@@ -160,21 +168,54 @@ - (IBAction)stepperChanged:(id)sender
     [self.proxy replaceValue:newValue forKey:@"value" notification:YES];
     
     if ([self.proxy _hasListeners:@"change"]) {
-        [self.proxy fireEvent:@"change" withObject:[NSDictionary dictionaryWithObject:newValue forKey:@"value"]];
+        [self.proxy fireEvent:@"change" withObject:[NSDictionary dictionaryWithObjectsAndKeys:
+                                                    [NSNumber numberWithDouble:stepper.value],@"value",
+                                                    [NSNumber numberWithDouble:stepper.minimumValue],@"minimum",
+                                                    [NSNumber numberWithFloat:stepper.maximumValue],@"maximum",
+                                                    nil]];
     }
 }
 
-- (IBAction)stepperTouch:(id)sender forEvent:(UIEvent *)event
+- (void)controlAction:(id)sender forEvent:(UIEvent *)event
 {
     UITouch *touch = [[event allTouches] anyObject];
-    if([self.proxy _hasListeners:@"click"]) {
-        [self.proxy fireEvent:@"click" withObject:[NSDictionary
-                                                   dictionaryWithDictionary:[TiUtils touchPropertiesToDictionary:touch
-                                                                                andPoint:[touch locationInView:self]]]];
+    NSString *fireEvent;
+    NSString * fireActionEvent = nil;
+    switch (touch.phase) {
+        case UITouchPhaseBegan:
+            if (touchStarted) {
+                return;
+            }
+            touchStarted = YES;
+            fireEvent = @"touchstart";
+            break;
+        case UITouchPhaseMoved:
+            fireEvent = @"touchmove";
+            break;
+        case UITouchPhaseEnded:
+            touchStarted = NO;
+            fireEvent = @"touchend";
+            if (stepper.highlighted) {
+                fireActionEvent = [touch tapCount] == 1 ? @"click" : ([touch tapCount] == 2 ? @"dblclick" : nil);
+            }
+            break;
+        case UITouchPhaseCancelled:
+            touchStarted = NO;
+            fireEvent = @"touchcancel";
+            break;
+        default:
+            return;
+    }
```

Comment:
Yeah we do , other wise the events don't get called. Its the same used inside the button class and the stepper are just 2 buttons inside a view as i mentioned above.